src/types.jl

```diff
@@ -615,13 +638,14 @@ end
     wkbNDR::GDAL.wkbNDR,
 )
 
-import Base.|
-
 for T in (GDALOpenFlag, FieldValidation)
     eval(quote
-        |(x::$T, y::UInt8)::UInt8 = UInt8(x) | y
-        |(x::UInt8, y::$T)::UInt8 = x | UInt8(y)
-        |(x::$T, y::$T)::UInt8 = UInt8(x) | UInt8(y)
```

Comment:
These lines should functionally still be there with two changes:
- Now also support `&`, not just `|`
- Now using `UInt32` instead of `UInt8` because some open flags do not fit into `UInt8`